WMF Framework/ExploreCardCollectionViewCell.swift

```diff
@@ -5,23 +5,35 @@ public protocol CardContent {
     func contentHeight(forWidth: CGFloat) -> CGFloat
 }
 
+fileprivate protocol CardBackgroundViewDelegate: AnyObject {
```

Comment:
My personal style, no need to update if you don't agree: When I'm writing code with nonobvious reasons to exist (what this does is obvious, but why it was written is only really discoverable if you dig into the commit/Phab history), I like to leave a 1 or 2 line comment explaining why it was added. Something to the effect of "Random Explore feed cards navigate to a specific random article when the card is tapped, and a truly random article when the card background (with `Another random article >` text) is tapped. This protocol allows the top of the background to navigate to the specific random article, as a user would expect."

Reply:
Yes, I prefer comments, but it seemed like the files I was working in didn't have any, so I thought that might be a preference. Will gladly add comments.